### ISSUE TITLE
Utility methods should return self

### ISSUE THREAD

**Issue by koliyo** [iamsalnikov/colorize#4]
Eg
```dart
  Colorize bgBlack() {
    return apply(Styles.BG_BLACK);
  }
```

**Comment by stepancar** [comment]
@koliyo you can just use cascade notation https://www.dartlang.org/guides/language/language-tour#cascade-notation-

**Comment by stepancar** [comment]
Do you have another reasons why you want to have itself return for all methods?

**Comment by koliyo** [comment]
Yes, forgot about the cascade operator, thanks for tip!

But it would still be nice for consistency, or else the `apply` could also return void, and always prefer using the cascade operator.

**Comment by stepancar** [comment]
@koliyo hm, agree with you. I have another question. May be we don't need public apply method?

**Comment by koliyo** [comment]
I don't think it is a problem with public `apply` method, it can definitely be useful, eg for calling parameterized instead of explicit direct calls to the wrappers.

**Comment by stepancar** [comment]
@koliyo, maybe let's close this issue?
